tests/acceptance/blockchain/conditions/test_conditions.py

```diff
@@ -0,0 +1,51 @@
+import json
+import pytest
+
+from nucypher.policy.conditions.lingo import ConditionLingo
+
+
+def test_erc20_evm_condition_evaluation(testerchain, evm_condition):
+    result, value = evm_condition.verify(provider=testerchain.provider, user_address=testerchain.unassigned_accounts[0])
+    assert result is True
+    result, value = evm_condition.verify(provider=testerchain.provider, user_address=testerchain.etherbase_account)
+    assert result is False
+
+
+@pytest.mark.skip('Need a way to handle user inputs like HRAC as context variables')
+def test_subscription_manager_condition_evaluation(testerchain, subscription_manager_condition):
+    hrac = None  # TODO
+    result, value = subscription_manager_condition.verify(provider=testerchain.provider, user_address=testerchain.unassigned_accounts[0], hrac=hrac)
+    assert result is True
+    result, value = subscription_manager_condition.verify(provider=testerchain.provider, user_address=testerchain.etherbase_account)
+    assert result is False
+
+
+def test_rpc_condition_evaluation(testerchain, rpc_condition):
+    address = testerchain.unassigned_accounts[0]
+    result, value = rpc_condition.verify(provider=testerchain.provider, user_address=address)
+    assert result is True
+
+
+def test_time_condition_evaluation(testerchain, timelock_condition):
+    address = testerchain.unassigned_accounts[0]
+    result, value = timelock_condition.verify(provider=testerchain.provider, user_address=address)
+    assert result is True
+
+
+def test_simple_compound_conditions_evaluation(testerchain):
```

Comment:
are we able to handle nested conidtions? (condition 1 AND condition 2) OR condition 3?

Reply:
Presently, no.